conbench/app/_plots.py

```diff
@@ -571,11 +580,14 @@ def time_series_plot(
         ),
     )
 
-    source_current_bm_mean = _source(
-        [dummy_hs_for_current_svs],
-        unit,
-        formatted=formatted,
-    )
+    # Edge case: this may be failed, in which case we cannot show a data point!
+    source_current_bm_mean = "failed"
+    if not current_benchmark_result.is_failed():
```

Comment:
Previously, we showed this data point, I think, despite the benchmark result being "failed".

Interesting, CC @jonkeane